Marlin/Marlin_main.cpp

```diff
@@ -2182,6 +2182,11 @@ static void clean_up_after_endstop_or_probe_move() {
     #if ENABLED(BLTOUCH)
       if (set_bltouch_deployed(true)) return true;
     #endif
+	
+    #if ENABLED(WAIT_FOR_BED_HEATER)
+      // Wait for bed to heat back up between probing points
+      while (thermalManager.isHeatingBed()) safe_delay(200);
+    #endif
```

Comment:
My prior caveat still stands. This behavior will sometimes be obnoxious. So we either need to check whether the bed was already heated up before invoking this delay, or we need to give the option a name that properly reflects this obnoxious aspect of its behavior.